py/client/pydeephaven/table.py

```diff
@@ -804,3 +804,79 @@ def delete(self, table: Table) -> None:
             self.session.input_table_service.delete(self, table)
         except Exception as e:
             raise DHError("delete data in the InputTable failed.") from e
+
+
+class MultiJoinTable:
+    """A MultiJoinTable is an object that contains the result of a multi-table natural join. To retrieve the underlying
+    result Table, use the table() method. """
```

Comment:
`table()` should use the linking syntax for the docs.

Reply:
```suggestion
    result Table, use the :meth:`~MultiJoinTable.table` method. """
```
I think this is the syntax.